src/test/java/ibm/jceplus/junit/openjceplusfips/TestECDSASignature.java

```diff
@@ -8,38 +8,17 @@
 
 package ibm.jceplus.junit.openjceplusfips;
 
-import junit.framework.Test;
-import junit.framework.TestSuite;
+import ibm.jceplus.junit.base.BaseTestECDSASignature;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
 
-public class TestECDSASignature extends ibm.jceplus.junit.base.BaseTestECDSASignature {
+@TestInstance(Lifecycle.PER_CLASS)
+public class TestECDSASignature extends BaseTestECDSASignature {
 
-    //--------------------------------------------------------------------------
-    //
-    //
-    static {
+    @BeforeAll
+    public void beforeAll() {
         Utils.loadProviderTestSuite();
+        setProviderName(Utils.TEST_SUITE_PROVIDER_NAME);
     }
-
-    //--------------------------------------------------------------------------
-    //
-    //
-    public TestECDSASignature() {
-        super(Utils.TEST_SUITE_PROVIDER_NAME);
-    }
-
-    //--------------------------------------------------------------------------
-    //
-    //
-    public static void main(String[] args) throws Exception {
-        junit.textui.TestRunner.run(suite());
-    }
-
-    //--------------------------------------------------------------------------
-    //
-    //
-    public static Test suite() {
-        TestSuite suite = new TestSuite(TestECDSASignature.class);
-        return suite;
-    }
-}
-
+}
```

Comment:
Add new line.

Reply:
New line added in commit bd27b562b46b98c872982261f4a7cb8bb2145955

Additionally a checkstyle rule was added to enforce this in the future.